docs/parsers/nextseq2k_nml.md

```diff
@@ -0,0 +1,62 @@
+# NextSeq2000
+
+The Software on the NextSeq2000 does not allow for automatic generation of a SampleSheet that includes a project column, so create a copy of the sample sheet called UploadList.csv and add the `Sample_Project` column ourselves.
+
+If you would like to use this parser you will also need to create this new file and `Sample_Project` column.
+
+## File Structure
+
+Please verify your file structure is correct.
+
+```
+.
+├── CopyComplete.txt
+├── UploadList.csv
+├── Analysis
+│   └── 1
+│       └── Data
+│           └── fastq
+│               ├── sample1_S1_L001_R1_001.fastq.gz
+│               ├── sample1_S1_L001_R2_001.fastq.gz
+│               ├── sample2_S1_L001_R1_001.fastq.gz
+│               ├── sample2_S1_L001_R2_001.fastq.gz
+│               ├── sample3_S1_L001_R1_001.fastq.gz
+│               └── sample3_S1_L001_R2_001.fastq.gz
+```
+
+Example files can be found: `/example_sample_sheets/miseq_run/`
```

Comment:
Not sure if we want to update this example file location.

Reply:
Good catch, thanks, I fixed that one up now.